Context/ConfiguredContextProvider.php

```diff
@@ -95,14 +95,16 @@ public function createConfiguredContext(ContextConfigurationInterface $config)
     }
 
     /**
-     * Add a context decorator to be apply to any generated context. First decorators provided are applied first.
+     * Add a context decorator to be apply to any generated context.
      *
      * @param ContextDecoratorInterface $decorator
-     * @return self
+     * @param int                       $priority
+     *
+     * @return $this
      */
-    public function addDecorator(ContextDecoratorInterface $decorator)
+    public function addDecorator(ContextDecoratorInterface $decorator, $priority = 500)
```

Comment:
You do, but I think it's more intuitive/ less surprising to use positive or negative numbers for that. Thinking f.e. of https://github.com/symfony-cmf/Routing/blob/master/ChainRouter.php#L80